lms/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -0,0 +1,142 @@
+        response = self._post_access_token_request(self.user, self.dop_client)
+        self.assertEqual(response.status_code, 200)
+        data = json.loads(response.content)
+        self.assertNotIn('refresh_token', data)
+
+
+@ddt.ddt
+class TestAuthorizationView(TestCase):
+    """
+    Test class for AccessTokenView
+    """
+
+    dop_adapter = adapters.DOPAdapter()
+    dot_adapter = adapters.DOTAdapter()
+
+    def setUp(self):
+        super(TestAuthorizationView, self).setUp()
+        self.user = UserFactory()
+        self.dot_app = self.dot_adapter.create_confidential_client(user=self.user, client_id='dot-app-client-id')
+        self.dop_client = self.dop_adapter.create_confidential_client(user=self.user, client_id='dop-app-client-id')
+
+    def test_authorization_view(self):
+        self.client.login(username=self.user.username, password='test')
+        response = self.client.post('/oauth2/authorize/', {
+            'client_id': self.dop_client.client_id,  # DOT is not yet supported (MA-2124)
+            'response_type': 'code',
+            'state': 'random_state_string',
+            'redirect_uri': 'http://example.edx/redirect',
+        }, follow=True)
+
+        self.assertEqual(response.status_code, 200)
+
+        # check form is in context and form params are valid
+        context = response.context  # pylint: disable=no-member
+        self.assertIn('form', context)
+
+    @skip("AuthorizationView dispatch is currently not hooked up to URL routing (MA-2124)")
+    def test_dot_authorization_view(self):
+        self.client.login(username=self.user.username, password='test')
+        response = self.client.post('/oauth2/authorize/', {
+            'client_id': self.dot_app.client_id,
+            'response_type': 'code',
+            'state': 'random_state_string',
+            'redirect_uri': 'http://example.edx/redirect',
+        }, follow=True)
+        self.assertEqual(response.status_code, 200)
+
+        # check form is in context and form params are valid
+        context = response.context  # pylint: disable=no-member
+
+        self.assertIn('form', context)
+        form = context['form']
+        self.assertEqual(form['redirect_uri'].value(), 'http://example.edx/redirect')
+        self.assertEqual(form['state'].value(), 'random_state_string')
+        self.assertEqual(form['client_id'].value(), self.dot_app.client_id)
+        self.assertFalse(form['allow'].value())
+
+    def test_dop_authorization_view(self):
+        self.client.login(username=self.user.username, password='test')
+        response = self.client.post('/oauth2/authorize/', {
+            'client_id': self.dop_client.client_id,
+            'response_type': 'code',
+            'state': 'random_state_string',
+            'redirect_uri': 'http://example.edx/redirect',
+        }, follow=True)
+        self.assertEqual(response.status_code, 200)
+
+        # check form is in context and form params are valid
```

Comment:
That is an obsolete comment.  form params are tested in test_authorization_view, above.  This just tests the parts that won't be true for DOT (once MA-2124 lands).  At that point, test_authorization_view should be @ddt enabled to test against both backends.